modules/reboot/src/main/java/org/mule/runtime/module/reboot/internal/DefaultMuleClassPathConfig.java

```diff
@@ -102,14 +112,11 @@ public void addFile(File jar) {
    * @return a list of {@link java.io.File}s
    */
   protected List<File> listJars(File path) {
-    File[] jars = path.listFiles(new FileFilter() {
-
-      public boolean accept(File pathname) {
-        try {
-          return pathname.getCanonicalPath().endsWith(".jar");
-        } catch (IOException e) {
-          throw new RuntimeException(e.getMessage());
-        }
+    File[] jars = path.listFiles((FileFilter) pathname -> {
+      try {
+        return pathname.getCanonicalPath().endsWith(".jar");
+      } catch (IOException e) {
+        throw new RuntimeException(e.getMessage());
```

Comment:
Isn't it better to create the RuntimeException with the IOException as a cause instead?